src/Mixer/Mixer.cpp

```diff
@@ -0,0 +1,109 @@
+#include "NAS2D/Mixer/Mixer.h"
+
+namespace NAS2D
+{
+
+	Mixer::Mixer(const std::string& name)
+	: mName(name)
+	{
+	}
+
+	Mixer::Mixer()
+	{
+	}
+
+	Mixer::~Mixer()
+	{
+	}
+
+	void Mixer::playSound(Sound& /*sound*/)

```

Comment:
I kind of like how you got around the Warning C4100 here.

Reply:
It's my go-to solution. It keeps the code self-documenting what with the header file still containing the argument names (and more importantly, by extension, Intellisense) and at the same time gets rid of the warning.